releasenotes/notes/add-estimator-qnn-270b31662988bef9.yaml

```diff
@@ -0,0 +1,56 @@
+---
+features:
+  - |
+    Introduced Estimator Quantum Neural Network
+    (class, `~qiskit_machine_learning.neural_networks.EstimatorQNN`) based on (runtime) primitives.
+    This implementation leverages the estimator primitive
+    (see :class:`~qiskit.primitives.BaseEstimator`) and the estimator gradients
+    (see :class:`~qiskit.algorithms.gradients.BaseEstimatorGradient`) to enable runtime access and
+    more efficient computation of forward and backward passes.
+
+    The new :class:`~qiskit_machine_learning.neural_networks.EstimatorQNN` exposes a similar
+    interface to the Opflow QNN, with a few differences.
+    One is the `quantum_instance` parameter. This parameter does not have a direct replacement,
+    and instead the `estimator` parameter must be used. The `gradient` parameter keeps the same name
+    as in the Opflow QNN implementation, but it no longer accepts Opflow gradient classes as inputs;
+    instead, this parameter expects an (optionally custom) primitive gradient.
+
+    The existing training algorithms such as :class:`~qiskit_machine_learning.algorithms.VQR`,
+    that were based on the Opflow QNN, are updated to accept both implementations. The implementation
+    of :class:`~qiskit_machine_learning.algorithms.NeuralNetworkRegressor` has not changed.
+
+    For example a `VQR` using `EstimatorQNN` can be trained as follows:
+
+
+    Example:
```

Comment:
I think these extra blank lines an Example:: can be deleted. With the prior sentence saying "for example ... as follows:" the outcome is not so great with it as you can see from the capture of the docs release note

![image](https://user-images.githubusercontent.com/40241007/198401879-21008e79-5db6-4771-8ab1-5736037e4b94.png)